stellargraph/core/graph.py

```diff
@@ -115,11 +116,36 @@ class StellarGraph:
 
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(
+        self,
+        graph=None,
```

Comment:
Does `graph` here need to be optional?
```suggestion
        graph,
```

Reply:
Yeah, it's currently truly optional:

```python
>>> import stellargraph as sg
>>> sg.StellarGraph()
NetworkXStellarGraph: Undirected multigraph
    Nodes: 0, Edges: 0
```